extract_i18n.sh

```diff
@@ -8,7 +8,7 @@ declare -a arr=(
     "eng_GB" "tha_TH" "ind_ID" "swa_TZ" "hin_IN" "fre_FR" "spa_CO" "spa_MX"
     "spa_AR" "mar_IN" "tel_IN" "por_PT" "mal_IN" "guj_IN" "tam_IN" "urd_IN"
     "urd_PK" "ben_IN" "ben_BD" "nep_NP" "fil_PH" "kan_IN" "ori_IN" "pan_IN"
-    "asm_IN")
+    "asm_IN", "km_KH" )
```

Comment:
I see according to gettext it's either: 

http://www.gnu.org/software/gettext/manual/gettext.html#Locale-Names
```
2.3.1 Locale Names
A locale name usually has the form ‘ll_CC’. Here ‘ll’ is an ISO 639 two-letter language code, and ‘CC’ is an ISO 3166 two-letter country code. For example, for German in Germany, ll is de, and CC is DE. You find a list of the language codes in appendix Language Codes and a list of the country codes in appendix Country Codes.
```

or this page from django:

https://docs.djangoproject.com/en/1.8/topics/i18n/

```
locale name
A locale name, either a language specification of the form ll or a combined language and country specification of the form ll_CC. Examples: it, de_AT, es, pt_BR. The language part is always in lower case and the country part in upper case. The separator is an underscore.
```

Both point to a two letter version of khmer... let me know?